Fixtures/iOS/Project.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		0403345F1F8F57CB0079CCF5 /* Model.xcdatamodeld in Sources */ = {isa = PBXBuildFile; fileRef = 0403345D1F8F57CB0079CCF5 /* Model.xcdatamodeld */; };
+		0403345F1F8F57CB0079CCF5 = {isa = PBXBuildFile; fileRef = 0403345D1F8F57CB0079CCF5; };
```

Comment:
Why are these changes necessary?

Reply:
Nop, I've reverted the change. Good catch 👍